.github/workflows/pull-request.yml

```
@@ -57,10 +57,10 @@
 
       - name: Check for linting errors in modified files
         if: steps.changed-files.outputs.only_changed != 'true'
-        env: 
+        env:
           CHANGED_FILES: ${{ steps.changed-files.outputs.all_changed_files }}
-        run: npx eslint ${CHANGED_FILES}
-        
+        run: npx eslint "${CHANGED_FILES}""
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC1009:info:1:1: The mentioned syntax error was in this simple command

(shellcheck)